docs/contributing-integrations.rst

```diff
@@ -205,3 +205,18 @@ are not yet any expected spans stored for it, so we need to create some.
 If in the process of writing tests for your integration you create a sample application,
 consider adding it to the `trace examples repository <https://github.com/Datadog/trace-examples>`_ along
 with screenshots of some example traces in the PR description.
+
+What does a complete PR look like when adding a new integration?
```

Comment:
May be worth a quick mention that as there are already many integrations built, pulling from them for example can be quite helpful, but the checklist looks good!

Reply:
Yeah I was gonna link to an example PR (maybe my dramatiq one), since it's hard to see all of the necessary changes in one single pane (which is what I was looking for when I wrote my dramatiq integration)